docs/en/sql-reference/table-functions/fuzzJSON.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: /en/sql-reference/table-functions/fuzzJSON
+sidebar_position: 75
+sidebar_label: fuzzJSON
+---
+
+# fuzzJSON
+
+Perturbs a JSON string with random variations.
+
+``` sql
+fuzzJSON(['source'[, 'random_seed'[, 'reuse_output'[, 'max_output_length']]]])
+```
+
+**Arguments**
+
+- `source` (String) - The source string representing structured data in JSON format.
+- `random_seed` (UInt64) - Specify the random seed manually to produce stable results. If NULL, the seed is randomly generated.
+- `reuse_output` (bool) - Reuse the output from a fuzzing process as input for the next fuzzer.
```

Comment:
Why we may want to have `reuse_output = false` ? This argument can seem a bit specific, do we really need to expose it?

But I don't have a strong opinion on this matter, so feel free to keep it if you consider it useful.